utils/preprocess_video_to_npy.py

```diff
@@ -0,0 +1,118 @@
+import ffmpeg
+import numpy as np
+import os
+import tyro
+import multiprocessing as mp
+from dataclasses import dataclass
+import json
+
+
+@dataclass
+class Args:
+    target_width, target_height = 160, 90
+    target_fps = 10
+    input_path: str = (
+        "/hkfs/work/workspace/scratch/tum_ind3695-jafa_ws_shared/data/knoms/"
```

Comment:
maybe we could have a more general default path here?
```suggestion
        "data/videos/"
```

Reply:
Yes